web/src/components/organisms/Settings/Members/hooks.ts

```diff
@@ -73,7 +96,8 @@ export default ({ workspaceId }: Props) => {
             }
           : undefined,
       )
-      .filter((user): user is Member => !!user);
+      .filter((user): user is Member => !!user)
+      .sort((a, b) => stringSortCallback(a.userId, b.userId));
```

Comment:
a and b don't have any meaning that I can see. Can you rename to user1 and user2?

Reply:
Fixed.